.vscode/schema/manifest-shim.json

```diff
@@ -0,0 +1,18 @@
+{
```

Comment:
This file should be maintained in https://github.com/DimensionDev/webextension-polyfill as one of its public API.